wsgi/Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN addgroup -S uwsgi && adduser -S -H -G uwsgi uwsgi && \
 
 COPY mime.types /etc/mime.types
 
-CMD ["/app/venv/bin/uwsgi", "--http-socket", ":8888", "--master", "-w", "application:application"]
+CMD ["/app/venv/bin/uwsgi", "--http-socket", ":8888", "--processes", "2", "--enable-threads", "--buffer-size", "16384", "--single-interpreter", "--master", "-w", "application:application"]
```

Comment:
These values should match the ones in https://github.com/Crown-Commercial-Service/digitalmarketplace-docker-base/blob/main/uwsgi.conf. Missing options should also be added.